src/components/button/button.css

```diff
@@ -0,0 +1,10 @@
+button {
```

Comment:
I would create a specific class for this button. Maybe call it `action-button`. You are currently styling all HTML button elements this way and I don't think you'd want that.